app/src/main/res/layout/configure_parking_lots_dialog_fragment_main_activity.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/dialog_fragment_configure_parking_lots_padding_size_xsmall"
+    tools:context=".activity.MainActivity">
+
+    <TextView
+        android:id="@+id/text_set_parking_dialog_fragment_title"
+        style="@style/baseTextStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/fragment_configure_parking_lots_title"
+        app:layout_constraintStart_toStartOf="parent"
```

Comment:
if you use match parent constraint start is useless

Reply:
constraints deleted